src/details/ArborX_DetailsDistributor.hpp

```diff
@@ -288,14 +288,11 @@ class Distributor
     bool const permutation_necessary = _permute.size() != 0;
     if (permutation_necessary)
     {
-      auto dest_buffer = ExportViewWithoutMemoryTraits(
+      ExportViewWithoutMemoryTraits dest_buffer(
           Kokkos::view_alloc(
-              space,
+              Kokkos::WithoutInitializing, space,
```

Comment:
```suggestion
              space, Kokkos::WithoutInitializing,
```
just to align it with the rest of the code in ArborX.